session/src/main/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandler.java

```diff
@@ -99,6 +99,8 @@ public APIGatewayProxyResponseEvent handleRequest(
             SessionItem auditSessionItem = new SessionItem();
             auditSessionItem.setSessionId(sessionId);
             auditSessionItem.setSubject(sessionRequest.getSubject());
+            auditSessionItem.setPersistentSessionId(sessionRequest.getPersistentSessionId());
```

Comment:
Update `SessionHandlerTest.java` to cover the new code here

Reply:
New fields added to the test shouldCreateAndSaveAddressSession.